lib/error.js

```diff
@@ -131,12 +131,17 @@ function isRetryableError(error) {
  * @class
  * @param {Error|string|object} message The error message
  * @property {string} message The error message
+ * @property {object} result The result document (provided if ok: 1)
```

Comment:
Please wrap the name of the argument with brackets to indicate it is optional, and add it to the `@param` documentation.